src/Widget/ItemList.php

```diff
@@ -106,27 +108,27 @@ public function getItemLayout($item): ItemLayout
     }
 
     /**
-     * Get message to show if the list is empty
+     * Get message content to show if the list is empty
      *
-     * @return string
+     * @return ValidHtml
      */
-    public function getEmptyStateMessage(): string
+    public function getEmptyStateMessage(): ValidHtml
     {
         if ($this->emptyStateMessage === null) {
-            return t('No items found.');
+            return new Text(t('No items found.'));
         }
 
         return $this->emptyStateMessage;
     }
 
     /**
-     * Set message to show if the list is empty
+     * Set message content to show if the list is empty
      *
-     * @param string $message
+     * @param ?ValidHtml $message
      *
      * @return $this
      */
-    public function setEmptyStateMessage(string $message): self
+    public function setEmptyStateMessage(?ValidHtml $message): self
```

Comment:
Use the `mixed` type (yeah, it's php 8 only but we raise the min requirement soon anyway) and use `Html::wantHtml` inside.